benchmarks/benchmark_base.py

```diff
@@ -239,7 +248,9 @@ def packet(self):
 
     @property
     def verysimple_packet_collection(self):
-        return self.nb_simulation_verysimple.transport.transport_state.packet_collection
+        return (
+            self.nb_simulation_verysimple.transport.transport_state.packet_collection
```

Comment:
why are we returning in 3 lines?

Reply:
These are changes suggested by black.

Reply:
I think the formatter that is suggested to be used is ruff. I might be wrong.

cc: @andrewfullard

Reply:
Ruff still needs to be implemented in TARDIS workflows.